### DISCUSSION TITLE
Lock card, open and closed states

### DISCUSSION THREAD

**Discussion by evoke0** [piitaya/lovelace-mushroom#1146] [Ideas]
Hello, I just started using the lock card with my nuki, it works like a charm and is exactly how it should be. However, is there any possibility to show further information about what the door state is? e.g. When the door is unlocked but closed, show: Unlocked - Closed, and viceversa, when the door is unlocked AND open, show: Unlocked - Open.

I understand this needs another sensor, which I have, how would that be implemented?
Thanks in advance.

**Comment by piitaya** [comment]
Hello. You must add another card for sensor state. Mushroom cards only support one entity.
